tests/README.md

```diff
@@ -39,4 +28,16 @@ to suite the tests need. Refer to the `tests/framework/README.md` for more detai
 IMPORTANT NOTES:
  * NEVER FORGET TO DESTROY A VKINSTANCE THAT WAS SUCCESSFULLY CREATED.
    * The loader loads dynamic libraries in `vkCreateInstance` and unloads them in `vkDestroyInstance`. If these dynamic libraries aren't unloaded, they leak state
-   into the next test that runs, causing spurious failures or even crashes.
\ No newline at end of file
+   into the next test that runs, causing spurious failures or even crashes.
+   * Use InstWrapper helper as well as DeviceWrapper to automate this concern away.
+
+## Using a specific loader with the tests
+
+The environment variable `VK_LOADER_TEST_LOADER_PATH` can be used to specify which vulkan-loader binary should be used. This is useful when writing tests to exercise
+a bug fix. Simply build the loader without the fix, stash it in a known location. Write the fix and the test that should exercise the bug and it passes. Then run the
```

Comment:
Nit:  Each sentence should always start on it's own line to ease the burden of merging changes from multiple peopal